src/components/EventCard.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import { CardContent, Typography } from '@material-ui/core';
```

Comment:
Can you restructure this to import each component separately? See [https://material-ui.com/guides/minimizing-bundle-size/#development-environment](https://material-ui.com/guides/minimizing-bundle-size/#development-environment)